tsconfig.json

```diff
@@ -1,18 +1,13 @@
 {
-	"include": ["./src/**/*.ts", "./src/**/*.vue"],
+	"extends": "@vue/tsconfig/tsconfig.json",
+	"include": ["src/**/*.ts"],
 	"exclude": ["node_modules", "vendor"],
 	"compilerOptions": {
 		"outDir": "./js",
 		"allowJs": true,
 		"checkJs": true,
 		"allowImportingTsExtensions": true,
 		"lib": ["ESNext"],
-		"module": "ESNext",
-		"moduleResolution": "bundler",
-		"verbatimModuleSyntax": true,
-		"target": "ES2020",
-		"esModuleInterop": true,
-		"forceConsistentCasingInFileNames": true,
```

Comment:
What was wrong here?

Reply:
covered by default config: `"extends": "@vue/tsconfig/tsconfig.json"`

Also we rolback and disabled checking for .vue files later, as it keep complaining on Windows + VSCode. Please check from your side, if there is something we've done wrong, when you have time

Reply:
This PR breaks building Talk Desktop and I cannot understand, how it does 👀